framework/doc/content/source/problems/EigenProblem.md

```diff
@@ -2,16 +2,109 @@
 
 !syntax description /Problem/EigenProblem
 
-The `EigenProblem` holds the nonlinear eigen system and organizes the computation
-of the residual and Jacobian in the context of an eigen system.
+`EigenProblem` is the primary eigenvalue system in the MOOSE framework.
+It offers linear and nonlinear eigensolvers by leveraging [SLEPc](https://slepc.upv.es)
+capabilities (mainly the EPS system). For linear problems,
+the system supports one eigenvalue and multiple eigenvalues. For
+nonlinear problems, we currently support solving for the minimum
```

Comment:
```suggestion
nonlinear problems, we currently support solving for the smallest (in magnitude)
```

Reply:
I may be wrong on this

Reply:
We can call either way. I think your change is fine as well.